### PR TITLE
feat(docs): document templates and community submission

### PR DESCRIPTION
**Pull request by rektdeckard**
https://linear.app/livekit/issue/DEX-831/add-documentation-to-template-index

- Adds documentation around templates and the process for submitting a community template
- Updates existing templates with `display_name` and `attrs` properties to support upcoming UX changes to sandboxes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+1. Clone the git repository associated with the template to your `/tmp` directory or equivalent
+2. De-git the repository so you don't include the template's history in your project
+3. Copy the `.env.example` file to `.env.local`
+    - Automatically filling in any `LIVEKIT_` environment variables that are recognized
+    - Prompting you to fill in any additional variables that are not recognized, such as required API keys
+4. Move the directory from `/tmp` to the location you specified
+5. Run the `post_create` task from the `taskfile.yaml` and print any user setup instructions
+
+At this point, you're ready to start developing your application! Some templates work best with others -- for example, an AI agent template may pair well with a voice assistant frontend template. It all depends on what you want to build. The beauty of LiveKit is that the primitives are interchangeable!
```
Comment (kath0la) [inline README.md:37]: The en dash should be an em dash.

And personally, I would remove the exclamation points. Or at least limit it to only one per paragraph!

```suggestion
At this point, you're ready to start developing your application! Some templates work best with others --- for example, an AI agent template may pair well with a voice assistant frontend template. It all depends on what you want to build. The beauty of LiveKit is that the primitives are interchangeable!
```

**Review by kath0la — COMMENTED** [review]
All the bullet list items should end with punctuation.

Nice README! I noticed that the repo warns that it's not protected.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+All templates in this index can be bootstrapped using the [LiveKit CLI](https://docs.livekit.io/cloud/home/cli). Running the following command will allow you to choose and initialize a template:
+
+```bash
+lk app create
+```
+
+After choosing a template and name for your project, the CLI will execute the following steps:
+
+1. Clone the git repository associated with the template to your `/tmp` directory or equivalent
+2. De-git the repository so you don't include the template's history in your project
```
[the remark this answers is not in the document]
Comment (rektdeckard) [reply README.md:30]: It's one I've come across in common SWE parlance. It's also a tool that is pretty well-known: https://github.com/Rich-Harris/degit

File: README.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+At this point, you're ready to start developing your application! Some templates work best with others -- for example, an AI agent template may pair well with a voice assistant frontend template. It all depends on what you want to build. The beauty of LiveKit is that the primitives are interchangeable!
+
+If you need a refresher on the basics, be sure to head to our documentation and read the [Intro to LiveKit](http://docs.livekit.io/home/get-started/intro-to-livekit/).
+
+## Creating your own template
+
+### Repository structure
+
+Your template may be any kind of component, from a front-end for a livestreaming app to a
```
[the remark this answers is not in the document]
Comment (rektdeckard) [reply README.md:45]: Oh yea, trailed off mid-thought 🤣

**Comment by rektdeckard** [comment]
@kath0la thanks! fixed your commends and added branch protection on `main` 👍

**Pull request merged**